BinaryProtocol-32bit.md

```diff
@@ -0,0 +1,184 @@
+# Binary Protocol 32-bit (v0.1)
+
+## Introduction
+
+The Harp Protocol is a binary communication protocol created in order to facilitate and unify the interaction between different devices. It was designed with efficiency and ease of parsing in mind.
+
+The protocol is based on addresses. Each address points to a certain memory position available in the device. These positions are called registers. Each register is defined by a data type and some meaningful functionality attached to the data.
+
+The Harp Binary Protocol is commonly used for all exchanges between a Controller and a Device. The Controller can be a computer, or a server and the Device can be a data acquisition or actuator microcontroller.
+
+The available packets are:
+ * Command: Sent by the Controller to the Device. Command messages can be used to read or write the register contents.
```

Comment:
At some point we discussed changing this to "Request" no?

Reply:
Yes, but @filcarv was not convinced. Also we would need to update the original protocol terminology as well, so let's keep it for a future revision.